runtimes/dotnet-3.1/src/RuntimeResponse.cs

```diff
@@ -28,7 +27,7 @@ namespace DotNetRuntime
             Dictionary<string, object?> data,
             int statusCode = 200)
         {
-            Data = JsonSerializer.Serialize(data);
+            Data = data;
```

Comment:
I'm seeing different patterns being followed for different runtimes.
In some cases we're sending serialised strings, and in some we're sending the JSON object as it is. 

Please test all the runtimes if this is working